src/02-room-details.js

```diff
@@ -49,7 +66,35 @@ function getRoomByDinosaurName(dinosaurs, rooms, dinosaurName) {}
       "Kit Hopkins Education Wing"
     ]
  */
-function getConnectedRoomNamesById(rooms, id) {}
+function getConnectedRoomNamesById(rooms, id) {
+  let finalArr = [];
+  let tempArr = [];
+  let intoArr = false;
+  for (const room of rooms) {
+    if (room.roomId === id) {
+        tempArr = room.connectsTo;
+        break;
+    }   
+  }
+  if (tempArr.length === 0) {
+    return `Room with ID of '${id}' could not be found.`
+    
+  }
+  for (const tempRoom of tempArr) {
+    for (let i=0; i< rooms.length; i++) {
+      if (tempRoom === rooms[i].roomId) {
+        finalArr.push(rooms[i].name);
+        intoArr = true;
+        break;
+      }
+    }
+    if (intoArr === false) {
+      return `Room with ID of 'incorrect-id' could not be found.`
```

Comment:
This return can be more dynamic. If the id was `123abcd`, we would want the error message to read `Room with ID of '123abcd' could not be found.`